manifests/init.pp

```diff
@@ -446,6 +447,15 @@
     }
   }
 
+  case type3x($sshd_acceptenv_list) {
```

Comment:
you could just use `validate_array()`

Reply:
validate_array Deprecated: Will be removed in a future version of stdlib. See validate_legacy.